src/lib/data/languages.ts

```diff
@@ -41,7 +43,13 @@ export const languages = [
   },
 ];
 
-export const formatLanguage = (language: string): string => {
-  const name = languages.find(({ id }) => id === language)?.value;
+// type FormatLanguageProps = { id: string; value: string }[] | undefined;
+type FormatLanguageProps = IdValue[] | undefined;
+
+export const formatLanguage = (
+  language: string | undefined | null,
+  languagesList: FormatLanguageProps = languages,
```

Comment:
I don't know that a separate prop is necessary for this. I'd just do `languagesList: IdValue[] = languages`. Note that if you provide a default value, TypeScript is smart enough to know that `languagesList` is optional, you don't have to explicitly make the type `IdValue[] | undefined`.

Reply:
Ah ok that is good to know.